tests/url-generation.js

```diff
@@ -59,6 +60,84 @@ describe("URL generation", function () {
         expect(url).includes(`ik-s=`);
     });
 
+    it("Signed URL with é in filename", function () {
+      const encodedUrl = encodeStringIfRequired("https://ik.imagekit.io/test_url_endpoint/test_é_path_alt.jpg");
+      expect(encodedUrl).equal("https://ik.imagekit.io/test_url_endpoint/test_%C3%A9_path_alt.jpg");
+      const signature = getSignature({
+        privateKey: "test_private_key",
+        url: encodedUrl,
+        urlEndpoint: "https://ik.imagekit.io/test_url_endpoint",
+        expiryTimestamp: "9999999999",
+      });
+      const url = imagekit.url({
+        path: "/test_é_path_alt.jpg",
+        signed: true,
+      });
+      expect(url).equal(`https://ik.imagekit.io/test_url_endpoint/test_é_path_alt.jpg?ik-s=09a329f06a5106a8b9c43de8fb6a64948fff7c59`);
+      expect(url).includes(`ik-s=${signature}`);
```

Comment:
This is repeated since the whole URL is being asserted in the statement above.

Reply:
done